db/seeds.rb

```diff
@@ -1,28 +1,34 @@
 # frozen_string_literal: true
 
+exit if Rails.env.production?
```

Comment:
As in Decidim I would let to seed in production if desired:

```suggestion
exit unless !Rails.env.production? || ENV["SEED"]
```